src/cim/components/Agent/NoAgentsAlert.tsx

```diff
@@ -1,21 +1,29 @@
 import React from 'react';
 import { Alert, AlertVariant, List, ListComponent, ListItem } from '@patternfly/react-core';
 import { Link } from 'react-router-dom';
+import { useTranslation } from '../../../common/hooks/use-translation-wrapper';
 
-const NoAgentsAlert = () => (
-  <Alert variant={AlertVariant.warning} title="No available hosts were found" isInline>
-    The cluster can not be installed yet because there are no available hosts found. To continue:
-    <br />
-    <List component={ListComponent.ol}>
-      <ListItem>
-        Add hosts to an{' '}
-        <Link to="/multicloud/infrastructure/environments">infrastructure environment</Link>
-      </ListItem>
-      <ListItem>
-        Continue configuration by editing this cluster and utilize the newly added hosts
-      </ListItem>
-    </List>
-  </Alert>
-);
+const NoAgentsAlert = () => {
+  const { t } = useTranslation();
+  return (
+    <Alert variant={AlertVariant.warning} title="No available hosts were found" isInline>
+      {t(
+        'ai:The cluster can not be installed yet because there are no available hosts found. To continue:',
+      )}
+      <br />
+      <List component={ListComponent.ol}>
+        <ListItem>
+          {t('ai:Add hosts to an')}{' '}
+          <Link to="/multicloud/infrastructure/environments">
+            {t('ai:infrastructure environment')}
+          </Link>
```

Comment:
this should be done with `Trans` component to avoid constructing messages https://react.i18next.com/latest/trans-component